sidesheet/src/components/sidesheet.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button, Tabs } from "@equinor/eds-core-react";
+import { SideSheet } from "@equinor/fusion-react-side-sheet";
+
+import { useState } from "react";
+
+const Sidesheet = () => {
+  const [openSheet, setOpenSheet] = useState(false);
+  const [activeTab, setActiveTab] = useState(0);
+  return (
+    <div>
+      <Button onClick={() => setOpenSheet(true)}>Open the</Button>
```

Comment:
Jeg ønsker ikke å ha knappen for å åpne sidesheet her inne i sidesheet. Den bør vi ha inne i completion frontend repoet når vi implementerer punch sidesheet. Kunne du tatt dette bort?

setOpenSheet og openSheet bør være props som kommer inn til sidesheeten. 

Jeg ønsker også at aøt av hardcoded info til sidesheeten skal komme fra completion frontend. Det var sikkert lurt å ha det her under utvikling, men det bør tas bort og gjøres om til props felter nå før du merger PR

Dette inkluderer actions, tabs og en child component som skal være content til sidesheeten.

Reply:
Eller kanskje det er bedre å fikse på dette i en annen PR? Jeg vet ikke helt. Sakene du har jobbet med handler jo egentlig ikke om å lage sidesheet komponenten.

Reply:
Nei, siden saken ikke handlet om sidesheet komponenten, kopierte jeg bare koden fra punch sidesheet for å ha noe å åpne fra pakken 😄

Reply:
Oki, da kan vi ta endringene på det en annen gang!